### PR TITLE
Added coverage result one-liner to status check. Reads out of newCode…

### PR DESCRIPTION
**Pull request by ryankillory**
…CommitPercentage env var.

### CODE REVIEW

**Review by MaitreyaBuddha — COMMENTED** [review]
I'm going to refactor this per https://leanplum.atlassian.net/browse/PR-336 for separate commit status. We should probably just close this PR

**Review by MaitreyaBuddha — APPROVED** [review]
LGTM for what we discussed. Will merge once approved by @ryankillory (you just have to look at https://github.com/Leanplum/ghprb-plugin/pull/2/commits/6e04dcf3fdd33bc81a34803bf69ae7cba8fdac09 for my changes)

File: src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java
Comment (ryankillory) [inline src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java]: Why 1%?

File: src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java
Comment (ryankillory) [inline src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java]: Why not just do this in the same line as getting the env var?

File: src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java
Comment (MaitreyaBuddha) [reply src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java]: Per the agreement and roadmap we have all new code requiring > 0 % coverage. This should not be hardcoded, it should be a config, but this is a start in that direction. Plus we don't have to block on this result until we are ready.

File: src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java
Comment (MaitreyaBuddha) [reply src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java]: I don't expect all of these to work for you. If at least one works let's just go with that and forget the others:

Because I use the string for string concatenation in the line above this.
Because we don't need to make lines as complicated as possible, we need to make them as simple as possible.
Because it makes the code clearer to what is happening.
Because dereferencing multiple things on the same line is a bad idea, even if it's a static function it's a matter of consistency.

Comment (ryankillory) [reply src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java]: I figured you would provide at least half these answers lol.
ok fine

File: src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java
Comment (ryankillory) [reply src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java]: actually do we want to make this a double? and just do > 0?

Comment (MaitreyaBuddha) [reply src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java]: I think it's arbitrary. Right now the script returns 69% coverage for a PR with 0% coverage.

Comment (MaitreyaBuddha) [reply src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java]: What I mean is, we're going to have to tune this later. We shouldn't worry about it having a 0.99% potential error right now.

**Pull request merged**
